lexicons/app/bsky/notification/listNotifications.json

```diff
@@ -8,6 +8,14 @@
       "parameters": {
         "type": "params",
         "properties": {
+          "filter": {
```

Comment:
Wouldn't `reason` be less generic and reflect what is being filtered?

Reply:
yeah I like that 👍